src/actions/search.js

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {batchActions} from 'redux-batched-actions';
+
+import {Client4} from 'client';
+import {SearchTypes} from 'action_types';
+
+import {forceLogoutIfNecessary} from './helpers';
+import {logError} from './errors';
+import {getProfilesAndStatusesForPosts} from './posts';
+
+export function searchPosts(teamId, terms, isOrSearch = 'false') {
```

Comment:
it has to be a string, the bool values aren't recognized by the REST API in go, so.. I can pass a boolean and then convert that to a string, should I do that?